tests/unit_tests/test_credentials.js

```diff
@@ -388,27 +390,27 @@ switch (test) {
 //
```

Comment:
Our policy is not to use comments as version control. This section should be removed.

Reply:
Same as above, was already there from the "past". we should make them work before instead of removing them because i assume we need as much tests as possible :)

Reply:
Done, test_credentials refactored and some tests recovered